### ISSUE TITLE
Platypus: Missing level-one heading

### ISSUE THREAD

**Issue by eddybrando** [Qiskit/qiskit.org#2072]
According to an Axe a11y audit performed on `/course/ch-prerequisites/environment-setup-guide-to-work-with-qiskit-textbook`, the page is missing level-one heading.

---

https://dequeuniversity.com/rules/axe/4.3/page-has-heading-one?application=axeAPI

---

**User impact:** Moderate
**Disabilities affected:** Sighted Keyboard Users, Blind, Low Vision, Deafblind
**Requirement(s):** Best Practice
**WCAG Success Criteria:** N/A
